generators/app/index.js

```diff
@@ -70,7 +70,11 @@ module.exports = class extends Generator {
       .map(str => str.trim())
       .filter(Boolean);
     if (!testScripts.find(script => script.startsWith('jest'))) {
-      testScripts.push('jest');
+      testScripts.push(
+        this.props.coveralls && this.props.testEnvironment !== 'jsdom'
+          ? 'jest --coverage'
```

Comment:
We could also just always use `--coverage`. It's a little bit longer, but it's working fine in all cases.

Reply:
Sounds good!